src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,27 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTab, onTabSelected }) => {
```

Comment:
The prop `activeTab` should be renamed to `activeTabId` to match the task description: "The `Tabs` component should accept the following props: `activeTabId` - the `id` of the tab that should be active."